hack/run-wsu-ci-e2e-test.sh

```diff
@@ -28,6 +28,10 @@ TEST_DIR=$WMCO_ROOT/internal/test/wsu
 # Make gopath if doesnt exist
 mkdir -p $GOPATH
 
+# Seems ansible-playbook doesn't work with ansible.cfg
+mkdir -p /.ansible
+chmod 777 -R /.ansible
```

Comment:
If this works, it should go in `Dockerfile.tools`. Otherwise we will run into issues running locally.